### PR TITLE
[CI] Improve visibility for `docker run` commands in the CI

### PR DESCRIPTION
**Pull request by amenasria**
### What does this PR do?

This PR adds `>-` and line breaks to `docker run` commands in the CI (mainly windows steps).

### Motivation

It was suggested in #14473 ([here](https://github.com/DataDog/datadog-agent/pull/14473#discussion_r1059016712)) because we keep adding argument to the `docker run` command and decreasing visibility in the code.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by clamoriniere — APPROVED** [review]

**Review by bkabrda — APPROVED** [review]

**Review by val06 — APPROVED** [review]

File: .gitlab/source_test/ebpf.yml [hunk 1]
```diff
@@ -58,8 +58,17 @@
   script:
     - $ErrorActionPreference = "Stop"
     - '$_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content ; Write-Host "Running on instance $($_instance_id)"'
-    - !reference [.setup_python_mirror_win]
-    - docker run --rm -m 8192M -v "$(Get-Location):c:\mnt" -e AWS_NETWORKING=true -e SIGN_WINDOWS=true -e PY_RUNTIMES="$PYTHON_RUNTIMES" -e GOMODCACHE="c:\modcache" -e PIP_INDEX_URL=${PIP_INDEX_URL} 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/windows_1809_${ARCH}:$Env:DATADOG_AGENT_WINBUILDIMAGES c:\mnt\tasks\winbuildscripts\sysprobe.bat
+    - >-
```
Comment (val06) [inline .gitlab/source_test/ebpf.yml:61]: i think you need just the `>` character

File: .gitlab/source_test/windows.yml [hunk 2]
```diff
@@ -10,8 +10,21 @@
     - '$_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content ; Write-Host "Running on instance $($_instance_id)"'
     # we pass in CI_JOB_URL and CI_JOB_NAME so that they can be added to additional tags
     # inside JUNIT_TAR and then later used by datadog-ci
-    - !reference [.setup_python_mirror_win]
-    - docker run --rm -m 8192M -v "$(Get-Location):c:\mnt" -e CI_JOB_URL="${CI_JOB_URL}" -e CI_JOB_NAME="${CI_JOB_NAME}" -e AWS_NETWORKING=true -e SIGN_WINDOWS=true -e PY_RUNTIMES="$PYTHON_RUNTIMES" -e GOMODCACHE="c:\modcache" -e JUNIT_TAR="c:\mnt\junit-${CI_JOB_NAME}.tgz" -e VCPKG_BINARY_SOURCES="clear;x-azblob,${vcpkgBlobSaSUrl}" -e PIP_INDEX_URL=${PIP_INDEX_URL} 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/windows_1809_${ARCH}:$Env:DATADOG_AGENT_WINBUILDIMAGES c:\mnt\tasks\winbuildscripts\unittests.bat
+    - >-
```
Comment (val06) [inline .gitlab/source_test/windows.yml:13]: i think you need just the `>` character

File: .gitlab/source_test/ebpf.yml (hunk 1 above)
Comment (amenasria) [reply .gitlab/source_test/ebpf.yml]: Right, fixed in [1949156](https://github.com/DataDog/datadog-agent/pull/14899/commits/1949156bb0f6750def2e4aeae939f2c62b18e160) ! Thanks !

File: .gitlab/source_test/windows.yml (hunk 2 above)
Comment (amenasria) [reply .gitlab/source_test/windows.yml]: Right, fixed in [1949156](https://github.com/DataDog/datadog-agent/pull/14899/commits/1949156bb0f6750def2e4aeae939f2c62b18e160) ! Thanks !

File: .gitlab/choco_deploy.yml [hunk 3]
```diff
@@ -20,5 +20,10 @@ publish_choco_7_x64:
     - mkdir nupkg
     - copy omnibus\pkg\*.nupkg nupkg\
     - Get-ChildItem nupkg
-    - docker run --rm -v "$(Get-Location):c:\mnt" -e CHOCOLATEY_API_KEY=${chocolateyApiKey} 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/windows_1809_${ARCH}:$Env:DATADOG_AGENT_WINBUILDIMAGES c:\mnt\tasks\winbuildscripts\chocopush.bat
+    - >
```
Comment (iglendd) [inline .gitlab/choco_deploy.yml:23]: What - > means here? Multiline?

**Review by iglendd — APPROVED** [review]

File: .gitlab/choco_deploy.yml (hunk 3 above)
Comment (amenasria) [reply .gitlab/choco_deploy.yml:23]: Yes ! This SO post explains it really well: https://stackoverflow.com/a/21699210

**Pull request merged**